src/mlpack/methods/ann/layer/isrlu.hpp

```diff
@@ -0,0 +1,145 @@
+/**
+ * @file methods/ann/layer/isrlu.hpp
+ * @author Abhinav Anand
+ *
+ * Definition of the ISRLU activation function as described by Jonathan T. Barron.
+ *
+ * For more information, read the following paper.
+ *
+ * @code
+ * @article{
+ *   author  = {Carlile, Brad and Delamarter, Guy and Kinney, Paul and Marti, Akiko and Whitney, Brian},
```

Comment:
I think this line doesn't align with our 80 character line limit - https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping.